README.md

```diff
@@ -86,124 +96,195 @@ See the playground to interactively use this:
 
 ## Documentation
 
-First of all: Try out the playground if you want to test-drive this package and learn how to use it. You can also have a look [at the playground code](./playground/app.vue) to see how to use `nuxt-session` in your app.
+First of all: Try out the playground if you want to test-drive this package and
+learn how to use it. You can also have a look
+[at the playground code](./playground/app.vue) to see how to use `nuxt-session`
+in your app.
+
+The `nuxt-session` maintains sessions: Persistent data across different requests
+by the same client (or: "user"). To maintain these sessions, `nuxt-session` sets
+a cookie with a unique client id for the currently connected client. Then after
+the cookie is set, the client will be uniquely identifiable by the server as
+long as:
```

Comment:
Sprry to jump in here: I would ask that you do not apply autoformstting in the form of line breaks to this whole readme. It's quite a big formatting change and for things lile screen readers its helpful to have one sentence in one line.

(I'm assuming that this change happened mostly automatically)